HaikuPorter/Package.py

```diff
@@ -142,8 +142,9 @@ def isBuildableOnArchitecture(self, architecture):
 
 	def getStatusOnSecondaryArchitecture(self, architecture,
 			secondaryArchitecture):
-		# check the secondary architecture
-		if secondaryArchitecture:
+		# check the secondary architecture (but ANY takes precedence)
+		if (secondaryArchitecture
+			and not Architectures.ANY in self.recipeKeys['ARCHITECTURES']):
```

Comment:
I'm not sure if this change was actually needed, I'll check this again. I think this is what causes "any" packages now being tried to be built twice in buildmaster mode (see e.g. https://build.haiku-os.org/buildmaster/master/x86_gcc2/?buildrunDir=8186&viewMode=expanded).